public/app/features/integrated-alerting/components/AlertRules/AlertRules.tsx

```diff
@@ -20,12 +21,14 @@ const {
   severity: severityColumn,
   summary: summaryColumn,
   threshold: thresholdColumn,
+  actions: actionsColumn,
 } = columns;
 
 export const AlertRules: FC = () => {
   const styles = useStyles(getStyles);
   const [addModalVisible, setAddModalVisible] = useState(false);
   const [pendingRequest, setPendingRequest] = useState(false);
+  const [selectedAlertRule, setSelectedAlertRule] = useState<AlertRule>();
```

Comment:
Call `setSelectedAlertRule` with null or undefined when clicking the alert-rule-template-add-modal-button to avoid having a selected rule after editing the first time.

Reply:
Done